ext/storage_sources/azure_store/azure_connection.cpp

```diff
@@ -44,9 +46,15 @@ azure_connection::list_objects(std::vector<std::string> &objects) const
     return 0;
 }
 
+/* 
+ * Puts an object (blob) given a reference to the filename
+ */
 int
 azure_connection::put_object(const std::string &file_name) const
 {
+    std::filesystem::path p(file_name);
+    Azure::Storage::Blobs::BlockBlobClient blobClient = _azure_client.GetBlockBlobClient(p.filename());
```

Comment:
let's use auto here, since it looks quite complicated.

Reply:
resolved here 
https://github.com/wiredtiger/wiredtiger/pull/8718/commits/ed135883a94c5734254cb1154062cb548ec3a29e